website/app/components/doc/page/sidecar.hbs

```diff
@@ -0,0 +1,3 @@
+<div class='doc-page-sidecar'>
```

Comment:
Nit: Inconsistent quote style. (Uses single quotes instead of double)

Reply:
Yes, you're right. I have to do a few updates to include a `hero` section in a few pages, and I'll fix it.

(no nitpicking for me, btw)